### PR TITLE
Improvements to API documentation and examples

### PR DESCRIPTION
**Pull request by MiguelMedeiros**
Review all API docs.

### CODE REVIEW

File: frontend/src/app/components/api-docs/api-docs.component.ts [hunk 1]
```diff
@@ -2417,7 +3294,7 @@ responseSample: `{
 
         const currencies = await markets.getCurrencies();
         console.log(currencies);`,
-        curl: `curl -X GET "https://bisq.markets/api/markets/currencies/"`,
+        curl: `curl "https://bisq.markets/api/currencies/"`,
```
Comment (wiz) [inline frontend/src/app/components/api-docs/api-docs.component.ts:3297]:
```suggestion
        curl: `curl "https://bisq.markets/api/currencies"`,
```

File: frontend/src/app/components/api-docs/api-docs.component.ts [hunk 2]
```diff
@@ -2290,33 +3129,79 @@ responseSample: `{
...
   const myAddress = await addresses.getAddress({ address });
   console.log(myAddress);`,
         commonJS: `const { addresses } = bisqJS();
-        const address = 'B1DgwRN92rdQ9xpEVCdXRfgeqGw9X4YtrZz';
+        const address = 'Bbc1qq8h46exega5skk0k8c066vv47sv9wgz0lklkw9';
 
         const myAddress = await addresses.getAddress({ address });
         console.log(myAddress);`,
-        curl: `curl -X GET "https://mempool.space/api/address/:address"`,
+        curl: `curl "https://bisq.markets/api/address/Bbc1qq8h46exega5skk0k8c066vv47sv9wgz0lklkw9"`,
```
Comment (wiz) [inline frontend/src/app/components/api-docs/api-docs.component.ts:3141]: curl: `curl "https://bisq.markets/api/v1/address/Bbc1qq8h46exega5skk0k8c066vv47sv9wgz0lklkw9"`,

Comment (wiz) [inline frontend/src/app/components/api-docs/api-docs.component.ts:3141]:
```suggestion
    curl: `curl "https://bisq.markets/api/v1/address/Bbc1qq8h46exega5skk0k8c066vv47sv9wgz0lklkw9"`,
```

Comment (wiz) [inline frontend/src/app/components/api-docs/api-docs.component.ts:3141]:
```suggestion
        curl: `curl "https://bisq.markets/api/v1/address/Bbc1qq8h46exega5skk0k8c066vv47sv9wgz0lklkw9"`,
```

File: frontend/src/app/components/api-docs/api-docs.component.ts [hunk 3]
```diff
@@ -2472,7 +3349,7 @@ responseSample: `{
 
         const hloc = await markets.getHloc({ market });
         console.log(hloc);`,
-          curl: `curl -X GET "https://bisq.markets/api/markets/hloc/"`,
+          curl: `curl "https://bisq.markets/api/hloc/?market=BTC_USD"`,
```
Comment (wiz) [inline frontend/src/app/components/api-docs/api-docs.component.ts:3352]:
```suggestion
          curl: `curl "https://bisq.markets/api/hloc?market=BTC_USD"`,
```

File: frontend/src/app/components/api-docs/api-docs.component.html [hunk 4]
```diff
@@ -1324,256 +1324,257 @@ <h2>{{ network.val === '' ? 'Bitcoin' : network.val.charAt(0).toUpperCase() + ne
...
+            <ngb-panel id="bisqMarketsVolumes">
               <ng-template ngbPanelTitle>
-                <span>GET Market HLOC</span>
+                <span>GET Market Volumes</span>
               </ng-template>
               <ng-template ngbPanelContent>
                 <div class="endpoint">
                   <div class="subtitle" i18n="Api docs endpoint">Endpoint</div>
-                  <a href="/bisq/hloc?market=xmr_btc" target="_blank">GET /bisq/api/hloc?market=xmr_btc</a>
+                  <a href="/bisq/volumes?basecurrency=BTC" target="_blank">GET /bisq/api/volumes?basecurrency=BTC</a>
```
Comment (wiz) [inline frontend/src/app/components/api-docs/api-docs.component.html:1459]: Simon says we should remove all references to `basecurrency` parameter for now

**Comment by wiz** [comment]
Please also use `document.location.hostname` in place of the hard-coded `mempool.space` and `bisq.markets`, since for example mempool.emzy.de or umbrel.local will be displayed for those users

**Comment by wiz** [comment]
For example when you visit http://bisqmktse2cabavbr2xjq7xw3h6g5ottemo5rolfcwt6aly6tp5fdryd.onion/api the API docs should display the onion from `document.location.hostname`

**Comment by MiguelMedeiros** [comment]
## Tasks
- [x] Add template to documentation.
- [x] Add support for BASE_MODULE=[mempool, bisq, liquid].
- [x] Add print results do CommonJS examples.
- [x] Add support for custom domains.
- [x] Remove basecurrency from `/volume` endpoint.

**Review by wiz — APPROVED** [review]

File: frontend/src/app/components/api-docs/api-docs.component.ts (hunk 1 above)
Comment (Franklincomp) [reply frontend/src/app/components/api-docs/api-docs.component.ts]: I am right in Franklin the rifle owner of these transactions I'm trying to gain my full access keep fraud off my case

**Pull request merged**